recipes/rosinstall_generator/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "rosinstall_generator" %}
+{% set version = "0.1.18" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/ros-infrastructure/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 3d43d47bae69d55e4cd2bad210803a4fcc3a28f19a26c672c6ee73c4bf80673d
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - rosinstall_generator=rosinstall_generator.cli:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - catkin_pkg >=0.1.28
```

Comment:
You probably don’t need all of these in host. This should only be the dependencies required to run the setup.py script.

Generally I start with python and pip then add more if it fails.